Mathlib/Combinatorics/SimpleGraph/Hasse.lean

```diff
@@ -7,6 +7,7 @@ import Mathlib.Combinatorics.SimpleGraph.Prod
 import Mathlib.Data.Fin.SuccPred
 import Mathlib.Data.ZMod.Basic
 import Mathlib.Order.SuccPred.Relation
+import Mathlib.Tactic.Simps.Basic
```

Comment:
I don't think you need this import
```suggestion
```

Reply:
Oops, I was trying things and I forgot to delete it